src/View/PasswordQuickInput.ts

```diff
@@ -16,9 +16,9 @@
 
 import * as vscode from 'vscode';
 
-// NOTE ASCII characters have codes ranging from u+0000 to u+007f
-function containsNonAscii(str: string): boolean {
-  return !/^[\u0000-\u007f]*$/.test(str);
+// NOTE Printable ASCII characters have codes ranging from u+0000 to u+007f
+function containsOnlyPrintableAscii(str: string): boolean {
```

Comment:
From clean code's view,
I changed negation from the function to avoid 'double negation' 

> https://refactoring.com/catalog/removeDoubleNegative.html